### PR TITLE
Update ResonanceAudio.cs with GetApartament1Transform

### PR DESCRIPTION
**Pull request by mariapensko**

### CODE REVIEW

File: platforms/unity/UnityIntegration/Assets/ResonanceAudio/Scripts/ResonanceAudio.cs [hunk 1]
```diff
@@ -421,8 +429,13 @@ private class RoomProperties {
 #else
   private const string pluginName = "audiopluginresonanceaudio";
 #endif  // !UNITY_EDITOR && UNITY_IOS
+    [DllImport(pluginName)]
+    private static extern void GetApartment1Transform(float x, float y, float z,
```
Comment (twoz) [inline platforms/unity/UnityIntegration/Assets/ResonanceAudio/Scripts/ResonanceAudio.cs]: Shouldn't there also be a definition of this function in unity.cc?

Comment (twoz) [inline platforms/unity/UnityIntegration/Assets/ResonanceAudio/Scripts/ResonanceAudio.cs]: The function name is `Get` but it returns void and parameters are passed by value? Shouldn't it be named `Set` instead?

File: platforms/unity/UnityIntegration/Assets/ResonanceAudio/Scripts/ResonanceAudio.cs [hunk 2]
```diff
@@ -45,8 +45,16 @@ public static class ResonanceAudio {
   private static Transform listenerTransform = null;
 
 #if UNITY_EDITOR
-  /// Default audio mixer group of the renderer.
-  public static AudioMixerGroup MixerGroup {
+    public static void UpdateApartament1Transform(float x, float y, float z,
```
Comment (twoz) [inline platforms/unity/UnityIntegration/Assets/ResonanceAudio/Scripts/ResonanceAudio.cs:48]: Where is the caller of this?

File: platforms/unity/UnityIntegration/Assets/ResonanceAudio/Scripts/ResonanceAudio.cs (hunk 1 above)
Comment (mariapensko) [reply platforms/unity/UnityIntegration/Assets/ResonanceAudio/Scripts/ResonanceAudio.cs]: Yes, maybe you're right. I thought kind of from the other side, like the plugin wants to get the values, but it works the other way around, the Unity Editor sets the values for the plugin. I'll fix it in the next commit.

File: platforms/unity/UnityIntegration/Assets/ResonanceAudio/Scripts/ResonanceAudio.cs (hunk 2 above)
Comment (mariapensko) [reply platforms/unity/UnityIntegration/Assets/ResonanceAudio/Scripts/ResonanceAudio.cs:48]: It is the script attached to the apartament1 object in Unity scene. It is called in its Start method.

File: platforms/unity/UnityIntegration/Assets/ResonanceAudio/Scripts/ResonanceAudio.cs (hunk 1 above)
Comment (mariapensko) [reply platforms/unity/UnityIntegration/Assets/ResonanceAudio/Scripts/ResonanceAudio.cs]: There is, added in the previous push (the one without the pull request). I just realized now that the ResonanceAudio.cs should be updated not only in the SpatialAudioTest repo, but also here.

**Comment by mariapensko** [comment]
Fixed. Ready to merge.

File: platforms/unity/UnityIntegration/Assets/ResonanceAudio/Scripts/ResonanceAudio.cs (hunk 2 above)
Comment (twoz) [reply platforms/unity/UnityIntegration/Assets/ResonanceAudio/Scripts/ResonanceAudio.cs:48]: Ack

File: platforms/unity/UnityIntegration/Assets/ResonanceAudio/Scripts/ResonanceAudio.cs (hunk 1 above)
Comment (twoz) [reply platforms/unity/UnityIntegration/Assets/ResonanceAudio/Scripts/ResonanceAudio.cs]: Ack

**Review by twoz — APPROVED** [review]

**Pull request merged**